region/lowernorfair/east.json

```diff
@@ -960,7 +967,36 @@
           "to": [
             {
               "id": 3,
-              "requires": [ "canPassBombPassages" ],
+              "requires": [
+                {"or": [
+                  {"and": [
+                    "heatProof",
+                    {"or": [
+                      "canUseMorphBombs",
+                      {"and": [
+                        "canUsePowerBombs",
+                        {"cost": [
+                          {"ammo": [{
+                            "type": "PowerBomb",
+                            "count": 3
```

Comment:
There are ways around using 2 of those PBs. 1 can be avoided with Screw attack, and 1 can be avoided with either X-ray stand-up + ceiling clip, or Spring Ball.

Considerations like this are why I'm more in favor of leaving PB costs to their own task that focuses on them.

Reply:
I think I'll leave it as is for now to not clutter it too much.

Reply:
Could you put in a FIXME note about it, then?

Reply:
Alrighty